examples/99-advanced/magnetic-fields.py

```diff
@@ -0,0 +1,76 @@
+"""
+.. _magnetic_fields_example:
+
+Plot Electric and Magnetic Fields
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example demonstrates how PyVista can plot electric and magnetic
+fields.
+
+The modeling results are courtesy of `prisae <https://github.com/prisae>`_ and
+are from the `emg3d Minimum working example
+<https://emsig.xyz/emg3d-gallery/gallery/tutorials/minimum_example.html#sphx-glr-gallery-tutorials-minimum-example-py>`_.
```

Comment:
You could shorten this
<https://emsig.xyz/emg3d-gallery/gallery/tutorials/minimum_example.html#sphx-glr-gallery-tutorials-minimum-example-py>`_
to
<https://emsig.xyz/emg3d-gallery/gallery/tutorials/minimum_example.html>`_

Reply:
```suggestion
<https://emsig.xyz/emg3d-gallery/gallery/tutorials/minimum_example.html>`_.
```